update/update-version.md

```diff
@@ -1,139 +1,187 @@
 
-1. [Optional] If a schematics workspace job that was initiated through the script encounters an issue, you can undo any modifications made by running the script again with the -z flag. For example,
-    ```sh
-    bash schematics_update_v3.x.x_to_v4.x.x.sh -z
-    ```
-    A new schematics workspace job reverting the state back to its prior condition, which existed prior to the execution of the script. The script will use the `revert.json` file that got created when the script initially ran
+1.  Monitor the status of the job in the IBM Cloud [console](https://cloud.ibm.com/schematics/workspaces).
+    - When the job completes successfully, go to the next step.
+    - If the job fails, see [Reverting changes](#reverting-changes).
 
-1. After the successful upgrade to the newer release of the VSI module, it is safe to remove the temporary files generated by the script.
-    ```sh
-    rm moved.json revert.json
-    ```
+### Apply the changes in Schematics
+
+1. Update your code to consume version 4.x.x, and then update your Schematics workspace to the version of the code that contains the updated module. Click **Generate plan** and make sure none of the VSIs will be re-created. You should see changes only to names. No resources should be set to be destroyed or re-created.
+1. Click **Apply plan**.
 
-## Local Terraform State file
+    If the job is successful, follow the steps in [Clean up](#clean-up). If the job fails, see [Reverting changes](#reverting-changes).
 
-If you have both the code and the Terraform state file stored locally on your machine, you can utilize the update_v3.x.x_to_v4.x.x.sh script to avoid the recreation of Virtual Server Instances (VSIs).
+## Local Terraform
 
-1. Make sure you have all the dependencies to run the script.
-    - IBM Cloud CLI
-    - IBM Cloud CLI 'is' plugin
-    - Terraform CLI
-    - jq
+If you store both the Terraform code and state file locally, run the `update_v3.x.x_to_v4.x.x.sh` script locally.
+
+1. Set the IBM Cloud API key that has access to your VPCs as an environment variable by running the following command:
 
-1. Set the `IBMCLOUD_API_KEY` variable as an environment variable.
-    1. IBMCLOUD_API_KEY : This is an IBM Cloud API key, which can be used for accessing the VPC. For more information, see https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui
     ```sh
     export IBMCLOUD_API_KEY="<API-KEY>" #pragma: allowlist secret
     ```
 
-1. Retrieve the VPC IDs and region of any VPCs in which VSIs are deployed
-    1. VPC IDs can be retrieved from the terraform output by running `terraform output` or use the following jq command to parse them from the state:
+    Replace `<API-KEY>` with the value of your API key.
+
+1. Get the VPC IDs and the regions of any VPCs that the VSIs are deployed in
+
+    - Get the VPC IDs from the Terraform output by running the `terraform output` command. Or use the following `jq` command to parse the IDs from the state:
```

Comment:
seems to render with bullet point and an `i`

Reply:
👍 changed both to unordered list items (`-`) because the order is not important.